s3/pkg/datastore/ceph/ceph.go

```diff
@@ -34,22 +35,38 @@ import (
 	"github.com/webrtcn/s3client/models"
 )
 
+const sampleBucket string = "sample"
+
 type CephAdapter struct {
 	backend *backendpb.BackendDetail
 	session *s3client.Client
 }
 
 func (ad *CephAdapter) BucketDelete(ctx context.Context, in *pb.Bucket) error {
+	bucket := ad.session.NewBucket()
+	err := bucket.Remove(in.Name)
+	if err != nil {
+		log.Error("the Delete bucket failed in ceph service with err:%s", err.Error())
+		return err
+	}
+	log.Debug("The bucket:%s deleted successful in ceph", in.Name)
 	return nil
 }
 
 func (ad *CephAdapter) BucketCreate(ctx context.Context, input *pb.Bucket) error {
+	bucket := ad.session.NewBucket()
+	err := bucket.Create(input.Name, models.Private)
+	if err != nil {
+		log.Error("the create bucket failed in ceph service with err:%s", err.Error())
+		return err
+	}
+	log.Debug("The bucket:%s creation successful in ceph", input.Name)
```

Comment:
nit: following the same tense as above. -> The bucket :%s **created** **successfully**...

Reply:
@anvithks done with the changes